src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/upsertTranslation.hpp

```diff
@@ -0,0 +1,166 @@
+            size_t flatbufferSize = parser.builder_.GetSize();
+
+            const rocksdb::Slice translationSlice(reinterpret_cast<const char*>(buf), flatbufferSize);
+            rocksDbWrapper.put(resource, translationSlice);
+        }
+        else
+        {
+            std::cerr << "No payload data" << std::endl;
+        }
+    }
+
+    /**
+     * @brief Updates a translation entry in a rocksdb database.
+     *
+     * @param data JSON with the required information to create or update a translation entry.
+     * @param rocksDbWrapper rocksdb wrapper instance.
+     */
+    static void updateTranslationEntry(const nlohmann::json& data, Utils::RocksDBWrapper& rocksDbWrapper)
+    {
+        std::string translationEntryStr;
+        if (rocksDbWrapper.get(data.at("resource").get<std::string>(), translationEntryStr))
+        {
+            const std::string resource = data.at("resource").get<std::string>();
+            std::string flatbufferSchemaStr;
+            flatbuffers::LoadFile(FLATBUFFER_PATH.c_str(), false, &flatbufferSchemaStr);
+            flatbuffers::Parser parser;
+            parser.opts.strict_json = true;
+
+            bool valid = parser.Parse(flatbufferSchemaStr.c_str(), INCLUDE_DIRECTORIES);
+            if (!valid)
+            {
+                std::cerr << parser.error_ << std::endl;
+            }
+
+            std::string translationEntry;
+            flatbuffers::GenText(parser, translationEntryStr.c_str(), &translationEntry);
+
+            try
+            {
+                nlohmann::json translationEntryJSON = nlohmann::json::parse(translationEntry);
+                nlohmann::json translationEntryJSONPatched = translationEntryJSON.patch(data.at("operations"));
+                std::string translationEntryJSONPatchedStr = translationEntryJSONPatched.dump();
+                valid = parser.Parse(translationEntryJSONPatchedStr.c_str());
+
+                if (!valid)
+                {
+                    std::cerr << parser.error_ << std::endl;
+                }
+
+                uint8_t* buf = parser.builder_.GetBufferPointer();
+                size_t flatbufferSize = parser.builder_.GetSize();
+
+                const rocksdb::Slice translationSlice(reinterpret_cast<const char*>(buf), flatbufferSize);
+                rocksDbWrapper.put(resource, translationSlice);
+            }
+            catch (...)
+            {
+                std::cerr << "Couldn't update translation entry id: " << resource << std::endl;
+            }
+        }
+    }
+
+public:
+    /**
+     * @brief Creates or modifies a translation entry in a rocksdb database.
+     *
+     * @param data JSON with the required information to create or update a translation entry.
+     * @param rocksDbWrapper rocksdb wrapper instance.
+     */
+    static void upsertTranslationEntry(const nlohmann::json& data, Utils::RocksDBWrapper& rocksDbWrapper)
+    {
+        if (data.contains("data"))
```

Comment:
If "data" variable doesn't contain "data" field, should be printed an invalid data format error message to std::cerr (like in each item)?